package/upgrade/lib.sh

```diff
@@ -210,3 +210,54 @@ detect_upgrade()
 
   UPGRADE_PREVIOUS_VERSION=$(echo "$upgrade_obj" | yq e .status.previousVersion -)
 }
+
+upgrade_addon()
+{
+  local name=$1
+  local namespace=$2
+  local version=$3
+
+  version=$(cat ./addons/${name}.yaml | yq e .spec.version)
+
+  cat > addon-patch.yaml <<EOF
```

Comment:
@ibrokethecloud, I see this error:
```
2022-10-04T05:51:01.376850609Z stderr F cat: ./addons/vm-import-controller.yaml: No such file or directory
2022-10-04T05:51:01.535749767Z stderr F Error from server (NotFound): addons.harvesterhci.io "vm-import-controller" not found
```

We need to add the files in https://github.com/harvester/harvester/blob/master/package/upgrade/Dockerfile

Reply:
@bk201 let me have a quick look

Reply:
@bk201 this should now be present.